src/server/storage/storageAPI.js

```diff
@@ -30,7 +46,6 @@ const setup = (app: Router, gunPublic: StorageAPI, storage: StorageAPI) => {
    */
   app.post(
     '/user/add',
-    passport.authenticate('jwt', { session: false }),
```

Comment:
so we're not using login with temporary creds during signup anymore ?

Reply:
we all /user endpoints have automatically authentication. see '/user/*'